README_zh-CN.md

```diff
@@ -26,17 +25,27 @@
 [![codecov](https://codecov.io/gh/open-mmlab/mmtracking/branch/master/graph/badge.svg)](https://codecov.io/gh/open-mmlab/mmtracking)
 [![license](https://img.shields.io/github/license/open-mmlab/mmtracking.svg)](https://github.com/open-mmlab/mmtracking/blob/master/LICENSE)
 
+[📘Documentation](https://mmtracking.readthedocs.io/) |
+[🛠️Installation](https://mmtracking.readthedocs.io/en/latest/install.html) |
+[👀Model Zoo](https://mmtracking.readthedocs.io/en/latest/modelzoo.html) |
+[🆕Update News](https://mmtracking.readthedocs.io/en/latest/changelog.html) |
+[🤔Reporting Issues](https://github.com/open-mmlab/mmtracking/issues/new/choose)
```

Comment:
Please use the link of Chinese docs if we have.
I remember we have `Documentation, Installation, Model Zoo` in Chinese.

Reply:
got it